debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/CoreInfoCommand.java

```diff
@@ -110,6 +110,13 @@ public void run(String command, String[] args, Context context, PrintStream out)
 			/* Print VM service level info */
 			out.println("JAVA SERVICE LEVEL INFO");
 			out.println(ras.serviceLevel().getCStringAtOffset(0));
+			try {
+				String productName = ras.productName().getCStringAtOffset(0);
+				out.println("JAVA PRODUCT INFO");
+				out.println(productName);
+			} catch (Exception e) {
```

Comment:
This should be more specific, catching only `NoSuchFieldException`  (for cores where `productName` doesn't exist).

Reply:
This should also do an explicit null check:
```java
				U8Pointer productName = ras.productName();
				if (productName.notNull()) {
					out.println("JAVA PRODUCT INFO");
					out.println(productName.getCStringAtOffset(0));
				}
```